src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1170,4 +1172,124 @@ private function addRobotsIndexSection(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addHttpClientSection(ArrayNodeDefinition $rootNode)
+    {
+        $subNode = $rootNode
+            ->children()
+                ->arrayNode('http_client')
+                    ->info('HTTP Client configuration')
+                    ->canBeEnabled()
```

Comment:
I think we should auto-enable this when the `http-client` component is installed. That's consistent with other components, and makes it so that, *just* by forcing the user to install `http-client`, we can more-or-less safely know that the `http_client` service